chore: add a helper to test the project and make eaiser the process (#3725)
replicatedhq/kURL: CONTRIBUTING.md

    ```
    *NOTE: `install.sh` runs are idempotent, consecutive runs on changed spec will update kURL installation.*

### Customizing the spec to do test upgrades and installs

>    See that when we run the target `make build/sample` the spec config under [script/Manifest](scripts/Manifest) will be
>    replaced with the sample spec config under the [testdata/sample/config](./testdata/sample/Manifest). Therefore, 
>    if you would like to test other configurations then must: 

Reviewer: ```suggestion
   When you run the target `make build/sample`, the config spec under [script/Manifest](scripts/Manifest) will be replaced with the sample spec in [testdata/sample/Manifest](./testdata/sample/Manifest).
   If you would like to test other configurations you must: 
```

>    - have the spec configuration that you would like to test out in [script/Manifest](scripts/Manifest)

Reviewer: ```suggestion
   - replace the spec configuration with that which you would like to test in [script/Manifest](scripts/Manifest)
```

   - run `make clean` to clean the directories used

>    - ensure that you will call the makefile targets to build the bundles images,i.e:

Reviewer: ```suggestion
   - ensure that you call the makefile targets to build the bundle assets, i.e:
```

   ```bash
     # Here we are building the bundles for k8s 1.25.3 to target ubuntu 22.02 SO
     make build/packages/kubernetes/1.25.3/ubuntu-22.04
     make build/packages/kubernetes/1.25.3/images
   ```
   
   - build the addons tarball for the specific versions and untar them, i.e:

   ```bash
     # Here we are building the tarball for containerd version 1.6.9
     make dist/containerd-1.6.9.tar.gz && tar xzvf dist/containerd-1.6.9.tar.gz
   ```

>    - ensure that all is rysnced with the remote server before run `./install.sh`

Reviewer: ```suggestion
   - ensure that all is rsynced with the remote server before running `./install.sh`
```
